torch/onnx/utils.py

```diff
@@ -247,18 +247,22 @@ def _model_to_graph(model, args, f, verbose=False, training=False,
 
     graph = _optimize_graph(graph, operator_export_type)
 
+    input_and_param_names = [val.uniqueName() for val in graph.inputs()]
```

Comment:
Okay, after digging into the last problem, let's move these three new statements to the end of this method (i.e., `_model_to_graph`). Because we may also change the value names in _set_input_and_output_names. (Very rare case, when input_names conflicts with existing weights, and these weights won't be covered by input_names, and they will be renamed to something else, because their names are occupied by some other nodes covered by inputs)

Reply:
Good point. Fixed.